.github/workflows/build.yml

```
@@ -0,0 +1,75 @@
+name: Build and Deploy
+on: [push]
+# TODO: limit this flow to when we push to main
```

This should prevent deploying changes unless they've been merged to the `main` branch

```suggestion
name: Build and Deploy
on:
  push
    branches:
      - main
```